gluetool/tool.py

```diff
@@ -332,6 +333,40 @@ def check_options(self):
 
             sys.exit(0)
 
+        if Glue.option('list-eval-context'):
+            import tabulate
```

Comment:
@happz why you are using these late imports?

Reply:
Right, it could be on top level. Fixed.